tests/test_ptable.py

```diff
@@ -276,6 +276,13 @@ def test_ptable_heatmap_splits(hide_f_block: bool) -> None:
         for elem in Element
     }
 
+    # Also test different data types
+    data_dict["H"] = {"a": 1, "b": 2}.values()
+    data_dict["He"] = [1, 2]
+    data_dict["Li"] = np.array([1, 2])
+    data_dict["Be"] = 1
+    data_dict["B"] = 2.0
```

Comment:
thanks, this is good! but i think these lines should go in `TestDataPreprocessor`, no?

Reply:
`TestDataPreprocessor` already has these test cases, so I assume it might be good to duplicate these in a real plotter :)

Reply:
true, we can have them in both. it doesn't have the `dict_values` yet. i'll add those

https://github.com/janosh/pymatviz/blob/d391350c79a46149f6a8a5d8059e33dc7752aff7/tests/test_ptable.py#L41

Reply:
Was thinking the same, so 25f8d708782ec7cc6ac2a7c6db3f8aeb856b0b59 :)